examples/solverdummies/fortran/solverdummy.f90

```diff
@@ -19,17 +19,19 @@ PROGRAM main
   CALL precicef_action_write_iter_checkp(writeItCheckp)
 
   WRITE (*,*) 'DUMMY: Starting Fortran solver dummy...'
+  
   CALL getarg(1, config)
   CALL getarg(2, participantName)
-  CALL getarg(3, meshName)
 
   IF(participantName .eq. 'SolverOne') THEN
     writeDataName = 'dataOne'
     readDataName = 'dataTwo'
+    meshName = 'MeshOne'
```

Comment:
Why do you not use the same pattern in the C and C++ solverdummy? (see comment above)